tests/test_scaler.py

```diff
@@ -86,9 +86,11 @@ def test_scaler_functionality(self):
         str(sca)
 
         sca.stage()
-        sca.configure()
+        old, new = sca.configure({})
```

Comment:
Scaler has a custom `configure` method defined on it which always returns a pair of empty dicts. Gotta scrub out "rogue" ("rouge?") implementations of `configure` and add a non-trivial test.